tee-worker/ts-tests/vc.test.ts

```diff
@@ -25,9 +23,13 @@ describeLitentry('VC test', async (context) => {
         assert.equal(who, u8aToHex(context.defaultSigner[0].addressRaw), 'check caller error');
     });
     step('Request VC', async () => {
-        const vc = await requestVC(context, context.defaultSigner[0], aesKey, true, context.shard, {
-            A1: assertion.A1,
-        });
-        assert(vc !== u8aToHex());
+        for (const key in assertion) {
+            const eventData = await requestVC(context, context.defaultSigner[0], aesKey, true, context.shard, {
+                [key]: assertion[key as keyof Assertion],
+            });
+            assert(eventData![0] == u8aToHex(context.defaultSigner[0].addressRaw) && eventData![1] && eventData![2]);
+            const registry = (await context.substrate.query.vcManagement.vcRegistry(eventData![1])) as any;
+            assert.equal(registry.toHuman()!['status'], 'Active');
```

Comment:
I think there're much more contents to assert 😂 , including the general asserts (like JSON format validation, metadata validation,  signature validation, hash validation) and specific asserts

but maybe you want to do it in a separate PR
cc @zhizming-zhong

Reply:
we do here temporarily is to confirm the effectiveness of vc, and other functions such as whether vc is correct will be added later. At this point,  just running the requestVC process.